test/JoeMakerV4.test.ts

```diff
@@ -270,7 +273,7 @@ describe("joeMakerV4", function () {
 
       await this.zap.zapIn(this.avaxUsdc.address, { value: ethers.utils.parseEther("2") })
       await this.avaxUsdc.transfer(this.joeMakerV4.address, await this.avaxUsdc.balanceOf(this.dev.address))
-      await this.joeMakerV4.convert(this.usdc.address, this.wavax.address)
+      await this.joeMakerV4.convert(this.usdc.address, this.wavax.address, "100")
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.avaxUsdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect((await this.usdc.balanceOf(BAR_ADDRESS)) - barBalance).to.be.greaterThan(0)
```

Comment:
Again, I think we need better assertions here; the expectation is that devs receive an expected amount of JOE